### ISSUE TITLE
Support of grails 3

### ISSUE THREAD

**Issue by bdbogjoe** [tednaleid/groovy-template-grails-asset-pipeline#11]
Any plan to update your plugin to support grails 3 ?

**Comment by tednaleid** [comment]
No, sorry, I'm not currently using grails 3 (or actually grails 2) so no plans right now.  Feel free to fork the project and fix/release it for grails 3 if you'd find it useful.
